flake.nix

```diff
@@ -15,6 +15,21 @@
       # Nixpkgs instantiated for supported system types.
       nixpkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
     in {
+      packages = forAllSystems (system:{
+        # this custom build can be removed once https://github.com/cloudfoundry/bosh-bootloader/issues/596 is implemented.
+        bosh-bootloader = nixpkgsFor.${system}.buildGoModule rec {
+          name = "bosh-bootloader";
+          src = nixpkgsFor.${system}.fetchgit {
+            url = "https://github.com/cloudfoundry/bosh-bootloader";
+            rev = "refs/tags/v9.0.17";
```

Comment:
Quick question: didn't just v9.0.17 work?

Reply:
It actually works, thanks! 👍